main.go

```diff
@@ -118,15 +118,26 @@ func destroyDNS(c *conf, hc *http.Client) error {
 	req.Header.Add("Accept", "application/json")
 	req.Header.Add("Authorization", fmt.Sprintf("Basic %s", c.konsAPIKey))
 
-	_, err = hc.Do(req)
+	response, err := hc.Do(req)
 	if err != nil {
 		return err
 	}
+	defer response.Body.Close()
+
+	if response.StatusCode != 200 {
+		// if status is not 200, log it, but do not consider it as a service failure
```

Comment:
You should, the `/create` endpoint is actually a create and update endpoint. Ignoring the outcome of the delete is how we ended up with tunnel addresses that point to 3 - 4 boxes (instead of one).

I don't know how to solve this, but raising it for you to be aware.